.gitignore

```
@@ -17,3 +17,6 @@ org.eclipse.jdt.groovy.core.prefs
 
 # VS Code
 .vscode
+
+# Text Files
+dictionary.properties
```

The `**/` prefix is really needed to prevent committing the file in a subdirectory:
```suggestion
**/dictionary.properties
```